### PR TITLE
updated for latest paths and command line parameter requirements

### PR DESCRIPTION
**Pull request by stleary-cisco**
**Description**
Recently the cookbook scripts were moved to new directories. This commit updates the import directory paths. 

**Refactoring**
* A decision was made to use command line parameters instead of a constants file. The usage statements now include more helpful information. 
* The main() functions were renamed to match the functionality, to make it easier to invoke from another script vs command line.

**Testing done**
Confirmed all scripts execute successfully.

### CODE REVIEW

File: examples/ftd_api_scripts/deploy.py [hunk 1]
```diff
@@ -26,12 +26,9 @@ def deploy(host, port, user, passwd):
     '''
```
Comment (thsmall) [inline examples/ftd_api_scripts/deploy.py]: remove ''' :)

File: examples/ftd_api_scripts/deploy.py [hunk 2]
```diff
@@ -74,7 +71,8 @@ def deploy(host, port, user, passwd):
     import sys
 
     if len(sys.argv) != 5:
-        print("Usage: python cookbook_scripts/deploy.py host port user passwd")
```
Comment (thsmall) [inline examples/ftd_api_scripts/deploy.py:77]: I like these usage statments!

File: examples/ftd_api_scripts/deploy.py (hunk 1 above)
Comment (stleary-cisco) [reply examples/ftd_api_scripts/deploy.py]: Fixed

File: examples/ftd_api_scripts/deploy.py (hunk 2 above)
Comment (stleary-cisco) [reply examples/ftd_api_scripts/deploy.py:77]: Thanks!

File: examples/ftd_api_scripts/ips_access_rule.py [hunk 3]
```diff
@@ -70,7 +73,23 @@ def main():
...
+        return False
+    else:
+        return True
 
 if __name__ == '__main__':
-    main()
+    import sys
+
+    if len(sys.argv) != 6:
+        print("Usage: python ftd_api_scripts/ips_access_rule.py host port user passwd policy_name")
```
Comment (thsmall) [inline examples/ftd_api_scripts/ips_access_rule.py:86]: for policy_name, what should this look like from the command line?  
put an example here.

Comment (stleary-cisco) [reply examples/ftd_api_scripts/ips_access_rule.py:86]: Updated all 3 scripts that use policy_name with an example message

**Review by thsmall — APPROVED** [review]

**Pull request merged**